exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/advanced.ipynb

```diff
@@ -256,7 +248,7 @@
    "source": [
     "## Cleanup\n",
     "\n",
-    "This sections shows how you can use the `SLCT Manager` to clean up the artificats created by `exaslct`:\n",
+    "This sections shows how you can use the instance of `ScriptLanguageContainer` class to clean up the artificats created by `exaslct`:\n",
```

Comment:
```suggestion
    "This sections shows how you can use the instance of `ScriptLanguageContainer` class to clean up the artifacts created by `exaslct`:\n",
```

Reply:
BTW: I still feel some tingling from each passage trying to describe dealing with SLCs when desperately trying  to distinguish between 

* an actual SLC, or its representation as a docker-image, file, in the bucketfs, instantiated / activated, ...
* the implementation artifact Python `class ScriptLanguageContainer` or an instance of it
* one of the Git repos SLC-release and SLC